src/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -118,7 +132,44 @@ public void onAttach(Activity activity) {
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        Log_OC.i(TAG, "onCreateView() start");
+        View v = super.onCreateView(inflater, container, savedInstanceState);
+
+        // Setup FAB listeners
```

Comment:
The setup of FAB listeners is being done twice, with some slight differences: one here and another in ```onActivityCreatedView()```.

One of both should be removed, probably this one.